documentation/library_guide/kernel_templates/single_pass_scan.rst

```diff
@@ -195,6 +207,12 @@ The initial configuration may be selected according to these high-level guidelin
   compute cores is key for better performance. To allow sufficient work to satisfy all
   X\ :sup:`e`-cores [#fnote2]_ on a GPU, use ``param.data_per_workitem * param.workgroup_size ≈ N / xe_core_count``.
 
+- For large inputs that fully saturate compute cores, maximizing ``param.workgroup_size`` and ``param.data_per_workitem``
+  without spilling out of register memory results in best performance. The Intel® oneAPI DPC++ Compiler reports warnings
+  when register spillage occurs. This may be used alongside guidance provided in the
+  `oneAPI GPU Optimization Guide <https://www.intel.com/content/www/us/en/docs/oneapi/optimization-guide-gpu/2025-0/registers-and-performance.html>`_
```

Comment:
Could you update this link to point to the newest version at https://www.intel.com/content/www/us/en/docs/oneapi/optimization-guide-gpu/2025-2/overview.html?